django/utils/feedgenerator.py

```diff
@@ -223,7 +223,7 @@ def latest_post_date(self):
                     if latest_date is None or item_date > latest_date:
                         latest_date = item_date
 
-        return latest_date or datetime.datetime.now()
+        return latest_date or datetime.datetime.now(tz=timezone.utc)
```

Comment:
Could be useful to add tests with `@override_settings(USE_TZ=True)` and  `@override_settings(USE_TZ=False)` so someone doesn't come along and inadvertently make a regression.